### DISCUSSION TITLE
Add OctaviOs

### DISCUSSION THREAD

**Discussion by ahmedhridoy371** [ROM-builders/temporary#19203] [General]
OctaviOs Active again and its  stable now hopefully we will see OctaviOs on cirrus soon . thanks for this awesome server

stable? yes its stable now because i officially maintain OctaviOs for lavender (Redmi Note 7/7s) so i know 🙂 

here OctaviOs GitHub link: https://github.com/Octavi-Staging/android_manifest

**Comment by ahmedhridoy371** [comment]
@Apon77 still depends on you & team this Os addable or not me waiting for your answer. thanks a lot brother

**Comment by Apon77** [comment]
stable but why their org name saying staging?
